### ISSUE TITLE
For anyone who can't get the code to run, I made a walkthrough.

### ISSUE THREAD

**Issue by KefeiQian** [shadow2469/BiRSwinT#3]
Hope it comes in handy.

https://github.com/KefeiQian/BiRSwinT/blob/main/get_started.md

**Comment by ktiays** [comment]
Your getting started doc are mostly good to me. But we noticed that the pre-trained model file (`ckpt_epoch_44.pth`) you use is not correct and we have uploaded the [right one](https://huggingface.co/Pluto2469/BiRSwinT/blob/main/ckpt_epoch_44.pth) to Hugging Face. Would you mind opening a pull request to add the doc? Thank you!

**Comment by KefeiQian** [comment]
Of course! I will change the ckpt_epoch_44.pth and make a pull request today. Thank you so much!

**Comment by 323417** [comment]
@KefeiQian 

After I clone the code, put the dataset and 2 pretrained parameters files, run the run.sh (BiRSwinT-maintrain_valid.py) when the output folder is empty.

The original author you gave the accuracy in the training process is slowly increasing, but the first round of accuracy I run is very high (73.4 % as shown in the figure), I don't know if it is based on the training data.

![0418-issue-70](https://github.com/shadow2469/BiRSwinT/assets/115760650/8c1f9d9d-bed9-4258-a411-c0d3db0229e9)

**Comment by KefeiQian** [comment]
Sorry for the late reply. You can add me through WeChat. My username is as same as my GitHub name(all lowercase). @323417
